packages/frontend-api/src/Model/Store/OpeningHours/StoreOpeningHoursApiProvider.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Store\OpeningHours;
+
+use DateTimeImmutable;
+use Shopsys\FrameworkBundle\Model\Store\OpeningHours\StoreOpeningHoursProvider;
+use Shopsys\FrameworkBundle\Model\Store\Store;
+
+class StoreOpeningHoursApiProvider extends StoreOpeningHoursProvider
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Store\Store $store
+     * @return bool
+     */
+    public function isOpenNow(Store $store): bool
+    {
+        $now = new DateTimeImmutable(timezone: $this->displayTimeZoneProvider->getDisplayTimeZoneByDomainId($store->getDomainId()));
+
+        return $this->getOpeningHoursSetting($store)->isOpenAt($now);
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Store\Store $store
+     * @return \Shopsys\FrontendApiBundle\Model\Store\OpeningHours\OpeningHoursWithDateData[]
+     */
+    public function getFollowingWeekOpeningHours(Store $store): array
+    {
+        $openingHoursData = [];
+
+        $today = new DateTimeImmutable('today', timezone: $this->displayTimeZoneProvider->getDisplayTimeZoneByDomainId($store->getDomainId()));
```

Comment:
```suggestion
        $today = new DateTimeImmutable('today', $this->displayTimeZoneProvider->getDisplayTimeZoneByDomainId($store->getDomainId()));
```

timezone is a second argument, so named argument is unnecessary

Reply:
thanks, resolved in https://github.com/shopsys/shopsys/commit/b61c2061f9673808794d71538399a24e737ca785